src/plugins/explore/common/index.ts

```diff
@@ -5,7 +5,11 @@
 
 export const PLUGIN_ID = 'explore';
 export const PLUGIN_NAME = 'Discover';
-export const LOGS_VIEW_ID = 'logs';
+export const EXPLORE_FLAVOR_ID = {
+  LOGS: 'logs',
+  TRACES: 'traces',
+  METRICS: 'metrics',
+} as const;
```

Comment:
nitpick: why not enum?

Reply:
just preference, i find it hard to understand enum when they are transpiled to javascript. But if OSD uses enum everywhere i can change it